.github/workflows/publish.yaml

```
@@ -0,0 +1,26 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - clients/js/[0-9].[0-9]+.[0-9]+
+      - contracts/[0-9].[0-9]+.[0-9]+
+      - integrations/hardhat/[0-9].[0-9]+.[0-9]+
```

I think we traditionally put the leading `v` in front of the version numbers in oasis-core, oasis-sdk, cli and also sapphire-paratime repo, for example:
```
oa@matevz-oa:~/oasis-sdk$ git tag
client-sdk/go/v0.1.0
client-sdk/go/v0.2.0
client-sdk/go/v0.3.0
client-sdk/go/v0.4.0
client-sdk/go/v0.5.0
client-sdk/go/v0.5.1
client-sdk/go/v0.6.0
client-sdk/go/v0.6.1
contract-sdk/v0.3.0
runtime-sdk/v0.1.0
runtime-sdk/v0.2.0
runtime-sdk/v0.3.0
runtime-sdk/v0.4.0
runtime-sdk/v0.5.0
runtime-sdk/v0.6.0
runtime-sdk/v0.7.0
runtime-sdk/v0.7.1
runtime-sdk/v0.7.2
runtime-sdk/v0.7.3
```

Perhaps it would be a good idea to have this consistent.

```suggestion
      - clients/js/v[0-9].[0-9]+.[0-9]+
      - contracts/v[0-9].[0-9]+.[0-9]+
      - integrations/hardhat/v[0-9].[0-9]+.[0-9]+
```

---

Alright alright. Let's go for it moving forward 😄 

We only have a few old tags [here](https://github.com/oasisprotocol/sapphire-paratime/releases/tag/clients%2Fjs%2F1.1.1) and [there](https://github.com/oasisprotocol/sapphire-paratime/releases/tag/clients%2Fgo%2Fv0.9.1).